### PR TITLE
Alpha Tag

### PR DESCRIPTION
**Pull request by willsputra**
##### Description

Added alpha tag

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Refers/Fixes

Fixes: #685

### CODE REVIEW

File: app/assets/v2/css/base.css [hunk 1]
```diff
@@ -1126,20 +1126,17 @@ input.is-invalid {
 }
 
 #alpha{
-    color: #a78600;
+    color: #000000;
```
Comment (mkosowsk) [inline app/assets/v2/css/base.css]: Bookkeeping but think we are just using `color: black;' throughout the codebase. @the-lostonemc ?

File: app/assets/yge/css/main.css [hunk 2]
```diff
@@ -10,20 +10,17 @@ body {
   width: 100%;
 }
 #alpha{
-    color: #a78600;
+    color: #000000;
```
Comment (mkosowsk) [inline app/assets/yge/css/main.css]: Same as above

File: app/assets/yge/css/main.css [hunk 3]
```diff
@@ -10,20 +10,17 @@ body {
...
     background-color: #ffce05;
     border-radius: 4px;
     text-transform: uppercase;;
     font-size: 14px;
-    padding: 5px 7px;
-    font-weight: 100;
+    padding: 2px 7px;
+	font-weight: 100;
+	font-family: 'Muli', sans-serif;
```
Comment (mkosowsk) [inline app/assets/yge/css/main.css]: Indentation off on this line and previous

**Comment by mkosowsk** [comment]
@willsputra dropped some super minor comments for ya to take a look at when you get a moment :) Overall LGTM but I did not test. @thelostone-mc got some cycles? 👍🏻

@willsputra https://github.com/gitcoinco/web/issues/685#issuecomment-377286764 are the images per this comment up to date or has this changed a bit? If there are some changes please add updated pics to https://github.com/gitcoinco/web/issues/685 when you get a moment but if not disregard. Thanks!

Great work! :)

File: app/assets/v2/css/base.css (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/css/base.css]: Well I believe we had agreed to stick to the hex color, so as to keep it uniform throughout ! Else we end up with names in a few areas and hex code in the rest !

Comment (mkosowsk) [reply app/assets/v2/css/base.css]: Whoops! @willsputra please disregard :)

**Comment by mbeacom** [comment]
Per @willsputra  https://user-images.githubusercontent.com/19514207/38202017-ddf71d34-36cc-11e8-97b5-e4e5b8accd82.jpg

**Comment by thelostone-mc** [comment]
@willsputra rebase gone wrong ? :P There are 15 commits
Easiest : cherry pick the commits.  Lemme me know if you need help with that :)

**Comment by willsputra** [comment]
@thelostone-mc yeah might need your help sorry 😞 would have to improve my git knowledge!

**Comment by thelostone-mc** [comment]
Pinging you over slack !!

**Comment by thelostone-mc** [comment]
@mbeacom @mkosowsk Everything taken care off in one nice commit 😎 

x

**Comment by mkosowsk** [comment]
LGTM, this PR has earned the Mitch Kosowski Seal of Approval™ 💪🏻🤠👍🏻

@willsputra mind sharing with the community what technique you used to get down to 1 commit? 👍🏻

@thelostone-mc truly the community is in the presence of a Git master 😳

**Comment by willsputra** [comment]
with the guidance of Git master @thelostone-mc :
1. made a new branch
2. cherrypicked the chosen commits with `git cherry-pick`
3. squashed two commits to one with `git rebase -i HEAD~2`
4. reset and push the original branch with
`git checkout 685`
`git reset --hard foobar`    
`git push origin -f 685`

**Pull request merged**